src/material-experimental/mdc-chips/testing/chip-harness.ts

```diff
@@ -22,6 +27,11 @@ export class MatChipHarness extends ComponentHarness {
   static with<T extends typeof MatChipHarness>(this: T, options: ChipHarnessFilters = {}):
       HarnessPredicate<InstanceType<T>> {
     return new HarnessPredicate(MatChipHarness, options)
+      .addOption(
+        'hasAvatar',
+        options.hasAvatar,
```

Comment:
Is it that common of a use case to be able to query only chips that have an avatar? We don't have anything similar for the other icons that can be projected (e.g. the remove icon).

Reply:
it's probably not that common. removed